seatunnel-e2e/seatunnel-connector-v2-e2e/connector-influxdb-e2e/src/test/resources/log4j.properties

```diff
@@ -0,0 +1,22 @@
+#
```

Comment:
Remove this file.

This is the common configuration of e2e
https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-e2e/seatunnel-e2e-common/src/test/resources/log4j2.properties

Reply:
> Remove this file.
> 
> This is the common configuration of e2e https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-e2e/seatunnel-e2e-common/src/test/resources/log4j2.properties

thinks，i have fixed . PTAL